src/main/java/org/myrobotlab/framework/Instantiator.java

```diff
@@ -18,7 +18,7 @@ static public Object getNewInstance(Class<?>[] cast, String classname, Object...
     try {
       return getThrowableNewInstance(cast, classname, params);
     } catch (Exception e) {
-      log.warn("getNewInstance failed for {} {} {}", cast, classname, e.getClass().getCanonicalName());
+      log.info("getNewInstance failed for {} {} {}", cast, classname, e.getClass().getCanonicalName());

```

Comment:
This probably is an error or at least a warn level message.  If this comes up, I would think there is a bug somewhere that you want to know about?

Reply:
primarily this is used to "attempt" to get gui's, but when a gui is not been developed, the framework displays a empty frame with "X has no display" 
It will return null in this case, and the calling code will handle it appropriately, there is also a throwable version